bootstrap/container/logging.go

```diff
@@ -32,3 +35,47 @@ func LoggingClientFrom(get di.Get) logger.LoggingClient {
 
 	return loggingClient
 }
+
+type LogrusAdaptor struct {
+	lc logger.LoggingClient
+}
+
+func (f *LogrusAdaptor) Format(entry *logrus.Entry) ([]byte, error) {
+	// Implement your custom formatting logic here
+	return []byte(fmt.Sprintf("[%s] %s\n", entry.Level, entry.Message)), nil
+}
+
+func (f *LogrusAdaptor) Levels() []logrus.Level {
+	return logrus.AllLevels
+}
+func (f *LogrusAdaptor) Fire(e *logrus.Entry) error {
+	switch e.Level {
+	case logrus.DebugLevel:
+		f.lc.Debug(e.Message)
+	case logrus.InfoLevel:
+		f.lc.Info(e.Message)
+	case logrus.WarnLevel:
+		f.lc.Warn(e.Message)
+	case logrus.ErrorLevel:
+		f.lc.Error(e.Message)
+	case logrus.FatalLevel:
+		f.lc.Error(e.Message)
+	case logrus.PanicLevel:
+		f.lc.Error(e.Message)
+	}
+
+	return nil
+}
+
+func AdaptLogrusBasedLogging(dic *di.Container) {
+	l := LoggingClientFrom(dic.Get)
+	// Create a new logger instance
+	hook := &LogrusAdaptor{
+		lc: l,
+	}
+	logrus.AddHook(hook)
+	logrus.SetFormatter(&logrus.TextFormatter{
+		DisableColors: true,
+	})
+	logrus.SetOutput(io.Discard)
+}
```

Comment:
I don't think this belongs in the `container` package. This package is for setting & getting objects from the DIC.

How about putting it in `utils` package?

Reply:
this file/package/function was already here. 
```
func LoggingClientFrom(get di.Get) logger.LoggingClient {
```
I placed these here for that reason.  I'm happy to move the additions or the whole file if you want? I just put these "near" the same location as that function.

Reply:
LoggingClientFrom should and existing code should stay. Your should move since that are not dic container related. i.e. they are beyond the scope of `container`